spec/index.bs

```diff
@@ -2684,6 +2685,7 @@ enum GPUFeatureName {
     "float32-filterable",
     "clip-distances",
     "dual-source-blending",
+    "texture-compression-bc-sliced-3d",
```

Comment:
nit: For discoverability I think this should be right after `texture-compression-bc`. (the order is unobservable to JS)

Reply:
Done, thank you!